src/table/components/virtualized-table/Totals.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useLayoutEffect, memo, useMemo } from 'react';
+import { VariableSizeList } from 'react-window';
+import { TotalsProps } from './types';
+import { HEADER_HEIGHT, PAGINATION_HEIGHT } from './constants';
+import TotalsCell from './TotalsCell';
+import { getTotalsCellStyle } from '../../utils/styling-utils';
+
+const Totals = (props: TotalsProps) => {
+  const { layout, rect, forwardRef, columnWidth, pageInfo, theme, totals, paginationNeeded } = props;
+  const totalsStyle = useMemo(() => getTotalsCellStyle(layout, theme), [layout, theme.name()]); // eslint-disable-line react-hooks/exhaustive-deps
+
+  useLayoutEffect(() => {
+    forwardRef?.current?.resetAfterIndex(0, true);
+    forwardRef?.current?.scrollTo(0);
+  }, [layout, pageInfo, forwardRef, columnWidth]);
+
+  if (!totals.isVisible) {
+    return null;
+  }
+
+  let top = totals.atTop ? HEADER_HEIGHT : rect.height - HEADER_HEIGHT;
```

Comment:
For the future, note that HEADER_HEIGHT is not only a dependent on setting but also the table width, if it becomes more that one row. There is a variable on the context that stores the head height. It is not working great, it misses some updated.

Reply:
Not entirely sure what you are talking about here? The height of the header is currently a hard-coded value in virtual mode. But I'm guessing that it should be possible to change it, ex. via setting a different font size?

Reply:
yes exactly

Reply:
Will have to handle that once it supports changing font 👍

Reply:
yupp, mostly FYI